File: C++/random_password.cpp
#include <iostream>
#include <string>
#include <cstdlib>
#include <ctime>

using namespace std;

int main()
{
    // define character set of password
    string charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!@#$%^&*()_+-=[]";

    // seed the random number generator
    srand(static_cast<unsigned int>(time(nullptr)));

    // randomly generate a password of length 12
    string password;
    for (int i = 0; i < 12; ++i)
    {
        int index = rand() % charset.length();
        password += charset[index];
    }

    // output the generated password
    cout << "Randomly generated password: " << password << endl;

    return 0;
}

File: Python/random_password.py
import random

# define character set of password
charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!@#$%^&*()_+-=[]"

# randomly generate a password of length 12
password = ""
for i in range(12):
    index = random.randint(0, len(charset)-1)
    password += charset[index]

# output the generated password
print("Randomly generated password: " + password)
